test/image-blueprints-bootc/layer1-base/group3/rhel96-bootc-brew-ec-with-optional-fips.containerfile

```diff
@@ -0,0 +1,16 @@
+# {{- if env.Getenv "BREW_EC_RELEASE_VERSION" "" -}}
```

Comment:
this is checking if EC BREW RPMs exist, but we don't need this check because these RPMs are not needed to build this containerfile

instead, we need to check if the parent image exist